src/partials/languages/he.html

```diff
@@ -24,22 +24,41 @@
 {% macro t(key) %}{{ {
   "language": "he",
   "direction": "rtl",
-  "clipboard.copy": "העתק ללוח",
+  "announce.dismiss": "לא להציג את זה שוב",
+  "clipboard.copy": "העתקה ללוח",
   "clipboard.copied": "הועתק ללוח",
-  "edit.link.title": "ערוך דף זה",
-  "footer.previous": "קודם",
-  "footer.next": "הַבָּא",
+  "consent.accept": "לקבל",
+  "consent.manage": "לנהל הגדרות",
+  "consent.reject": "לדחות",
+  "edit.link.title": "עריכת הדף הזה",
+  "footer.previous": "הקודם",
+  "footer.next": "הבא",
+  "footer.title": "כותרת תחתונה",
+  "header.title": "כותרת עליונה",
   "meta.comments": "הערות",
-  "meta.source": "מָקוֹר",
+  "meta.source": "מקור",
+  "nav.title": "ניווט",
+  "search.config.lang": "he",
```

Comment:
Oh, also, `he` is not available as a search language. Please remove. Search should work without this separator.

Reply:
Done.